templates/footer.html

```diff
@@ -0,0 +1,5 @@
+				</div>
```

Comment:
Separating the theme is something that can be done in a different way. I suggest you open a new Pull Request where you have a `head.html` which only includes the content inside the `<head></head>` HTML element. That way changing the theme should be super easy. 

See, this is an idea I had and forgot to implement. Good job!